xbmc/LangInfo.cpp

```diff
@@ -53,7 +53,12 @@ static std::string shortDateFormats[] = {
   "DD.M.YYYY",
   "D.M.YYYY",
   "D. M. YYYY",
-  "YYYY.MM.DD"
+  "YYYY.MM.DD",
+  // short date formats with abbreviated month and 2 digit year
+  "D-mmm-YY",
+  "DD-mmm-YY",
+  "D mmm YY",
+  "DD mmm YY"
```

Comment:
@howie-f - I mimicked the formatting that was already in the code.

With regards the 'clang-format' stuff:  This was already not there so I did not add it.

With regards to the comma at the end of the last item:  The last item in the list had no comma before I made my change, so I did not add a comma.  I have also encountered issues in the past with some languages throwing a syntax/compiler error when encountering dangling commas at the end of lists, consequently, I tend to avoid them out of an abundance of paranoia.

As you can probably tell, I'm still new to GitHub and the processes that Team-Kodi have in place.  If you think that it is best to make these changes, please feel free to make them, I don't mind.  If, however, I, as the creator of the PR, am required to be the one make the changes so that they can be checked by another person, please advise that this is part of the process and I will make the changes.

Thanks for your guidance.

Reply:
The required changes have been made pushed to GItHub.

Reply:
nice, well done 👍 
as you can see the clang diff below is much cleaner https://jenkins.kodi.tv/job/BuildMulti-PR/26002/artifact/PR22969.diff
probably want to fix the remaining format issues?
it‘s the same procedure like you did

Reply:
that is what our [code guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md) are for and you checked the box

> My code follows the [Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md) of this project

there's much legacy code in the codebase that doesnt follow these guidelines, because they didn't exist right from start. for a pr to be accepted every new or touched code must follow. even if it's only 2 or 4 lines. it's always a good idea to make the code more readable and maintainable. _many pennies make a dollar_

just take a look at the jenkins diff https://jenkins.kodi.tv/job/BuildMulti-PR/26002/artifact/PR22969.diff . it tells you what needs to be done for this pr to be ready.

Reply:
At first it looked like not much had changes.  But then I realised that some items were moved by a space or two.  For example, this line:

```
-  dateTime3.SetDateTime(1991, 05, 9, 12, 34, 56);  //Need a single digit date
+  dateTime3.SetDateTime(1991, 05, 9, 12, 34, 56); //Need a single digit date
```
Are you saying that the code is non-compliant because of an extra space before a comment?

Reply:
> Are you saying that the code is non-compliant because of an extra space before a comment?

strictly speaking, yes.

Reply:
The changes required by Jenkins were applied yesterday.  What is the next step?